src/client/ClientDataResolver.js

```diff
@@ -244,7 +244,7 @@ class ClientDataResolver {
         if (/^https?:\/\//.test(resource)) {
           request.get(resource)
             .set('Content-Type', 'blob')
-            .end((err, res) => err ? reject(err) : resolve(res.body));
+            .end((err, res) => err ? reject(err) : resolve(res.body instanceof Buffer ? res.body : null));
```

Comment:
Should probably only resolve if it's a Buffer; otherwise reject
```js
end((err, res) => err ? reject(err) : res.body instanceof Buffer ? resolve(res.body) : reject(new Error('Body is not a Buffer')));
```

This may be too long, so it may be necessary to change the arrow function to the long form of it.